test/test.js

```diff
@@ -313,6 +313,32 @@ QUnit.test( "check(): simple", function( assert ) {
 	assert.ok( v.size() === 0, "No more errors" );
 } );
 
+QUnit.test( "numberOfInvalids(): count invalid fields with empty message", function( assert ) {
+	assert.expect( 2 );
+
+	var v = $( "#testForm23" ).validate( {
+		rules: {
+			box1: {
+				required: true
+			},
+			box2: {
+				required: true
+			}
+		}
+	} );
+
+	var requiredMsg = $.validator.messages.required;
+	$.validator.messages.required = "";
+
+	// Validate the form manually
+	v.form();
+
+	assert.equal( v.errorList.length, 2, "There is only one error" );
+	assert.equal( v.numberOfInvalids(), 2, "There is only one error" );
```

Comment:
Ops my bad, I forgot to update the message because I used to have only one error before adding another element to the list. I'll update it soon